src/documentation/pipelines/design-your-pipeline.md

```diff
@@ -98,37 +98,39 @@ The pipeline in Figure 3 performs the same operation in a different way - with o
 
 where each element in the input `PCollection` is processed once. See the example code below:
 ```java
-//define main stream and side output
-final TupleTag<String> mainStreamTag = new TupleTag<String>(){};
-final TupleTag<String> sideoutTag = new TupleTag<String>(){};
+// Define two TupleTags - one for the main output and one for the additional output.
+final TupleTag<String> mainOutTag = new TupleTag<String>(){};
```

Comment:
This could be something like 'startsWithATag' and 'startsWithBTag' or something that communicates more about what they're being used for.